src/Notebooks/Training_aztk.ipynb

```diff
@@ -49,7 +49,7 @@
    "outputs": [],
    "source": [
     "STORAGE_ACCOUNT_SUFFIX = 'core.windows.net'\n",
-    "STORAGE_ACCOUNT_NAME = 'stggfmfb5nj7ekn6'\n",
```

Comment:
this will make editing this notebook difficult.

Reply:
by that I mean the scenario when a data scientist updates the notebook and saves the updated version. it will work ok within one solution, but if they try to reuse an updated notebook across multiple deployments, things will break.